pkg/agent/agent.go

```diff
@@ -216,14 +216,6 @@ func (k *Klocksmith) watchUpdateStatus(update func(s updateengine.Status)) {
 
 	go k.ue.ReceiveStatuses(ch, stop)
 
-	// synthesize first message
```

Comment:
Also from discussion, we want to document the the initial special GetStatus call is to solve the specific scenario in which we missed a status message which will not be sent again. The calling code expects to get that message before regular passive receiving of statuses begins.

We plan to ignore any error on the initial GetStatus as there is not much else that can be done. We don't want to retry forever and do want continue on to start receiving statuses. In future I can see us maybe having an `initialFetch bool` option to receive statuses, but a comment would be good for now.